apps/learninglocker/templates/services/app/job_storage.yml.j2

```diff
@@ -20,6 +20,11 @@ spec:
         job_stamp: "{{ job_stamp }}"
         deployment_stamp: "{{ deployment_stamp }}"
     spec:
+{% set image_pull_secret_name = busybox_image_pull_secret_name or default_image_pull_secret_name %}
```

Comment:
```suggestion
{% set image_pull_secret_name = learninglocker_image_pull_secret_name or default_image_pull_secret_name %}
```

Reply:
This job uses the busybox image so I think this line was correct.